### PR TITLE
Feature/userprofile

### PR DESCRIPTION
**Pull request by muchendumaina**

### CODE REVIEW

File: src/components/topNav/topNav.jsx [hunk 1]
```diff
@@ -1,23 +1,22 @@
 import React from "react";
-import navStyle from './topNav.module.scss'
+import navStyle from "./topNav.module.scss";
```
Comment (riungemaina) [inline src/components/topNav/topNav.jsx:2]: Change prettier to use single quotes

File: src/pages/user/user.jsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
+import userStyle from "./user.module.scss";
+import emailImg from "./images/email.svg";
+import phoneImg from "./images/phone.svg";
+import passwordImg from "./images/password.svg";
+import profileImg from "./images/profile.svg";
+import nextImg from "./images/next.svg";
```
Comment (riungemaina) [inline src/pages/user/user.jsx:6]: Move all the images to elements so that they can be reused

File: src/pages/user/user.module.scss [hunk 3]
```diff
@@ -0,0 +1,203 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
```
Comment (riungemaina) [inline src/pages/user/user.module.scss:1]: Remove this, poppins already exists; it is imported at the top level see that here https://github.com/castynet-studios/pms-fe/blob/main/src/index.scss#L1

**Review by riungemaina — CHANGES_REQUESTED** [review]

**Comment by riungemaina** [comment]
Also update the profile picture

File: src/components/topNav/topNav.jsx (hunk 1 above)
Comment (riungemaina) [reply src/components/topNav/topNav.jsx:2]: https://github.com/castynet-studios/javascript#strings

File: src/pages/user/user.jsx (hunk 2 above)
Comment (muchendumaina) [reply src/pages/user/user.jsx:6]: Done

File: src/pages/user/user.module.scss (hunk 3 above)
Comment (muchendumaina) [reply src/pages/user/user.module.scss:1]: Done

**Comment by muchendumaina** [comment]
All changes made

File: src/elements/icons/icons.jsx [hunk 4]
```diff
@@ -1,18 +1,23 @@
...
-import { Ambulance } from '@styled-icons/fa-solid/Ambulance';
-import { MapPin4 } from '@styled-icons/remix-fill/MapPin4';
-import { Feedly } from '@styled-icons/simple-icons/Feedly';
-import { Spreadsheet } from '@styled-icons/boxicons-solid/Spreadsheet';
-import { Todo } from '@styled-icons/remix-fill/Todo';
-import { ChevronsRight } from '@styled-icons/boxicons-regular/ChevronsRight';
-import { ArrowheadRightOutline } from 'styled-icons/evaicons-outline';
-import { PiggyBank } from 'styled-icons/fa-solid';
-import { PatchPlus } from 'styled-icons/bootstrap';
+import { Copyright } from "@styled-icons/boxicons-regular/Copyright";
```
Comment (riungemaina) [inline src/elements/icons/icons.jsx:1]: Still using double quotes

File: src/pages/user/user.jsx [hunk 5]
```diff
@@ -1,33 +1,31 @@
 import userStyle from "./user.module.scss";
```
Comment (riungemaina) [inline src/pages/user/user.jsx:1]: Using double quotes

**Review by riungemaina — CHANGES_REQUESTED** [review]

**Review by riungemaina — APPROVED** [review]

**Pull request merged**